packages/core/src/resources/ProjectJobTokenScopes.ts

```diff
@@ -1,43 +1,115 @@
+  ): Promise<GitlabAPIResponse<JobTokenScopeSchema, C, E, void>> {
+    return RequestHelper.get<JobTokenScopeSchema>()(
+      this,
+      endpoint`${projectId}/job_token_scope`,
+      options,
+    );
+  }
 
-  show<E extends boolean = false>(
+  edit<E extends boolean = false>(
     projectId: string | number,
+    enabled: boolean,
     options?: Sudo & ShowExpanded<E>,
-  ): Promise<GitlabAPIResponse<JobTokenScopeSchema, C, E, void>>;
+  ): Promise<GitlabAPIResponse<JobTokenScopeSchema, C, E, void>> {
+    return RequestHelper.patch<JobTokenScopeSchema>()(
+      this,
+      endpoint`${projectId}/job_token_scope`,
+      { ...options, enabled },
+    );
+  }
 
   showInboundAllowList<E extends boolean = false>(
     projectId: string | number,
     options?: Sudo & ShowExpanded<E>,
-  ): Promise<GitlabAPIResponse<SimpleProjectSchema[], C, E, void>>;
+  ): Promise<GitlabAPIResponse<SimpleProjectSchema[], C, E, void>> {
+    return RequestHelper.get<SimpleProjectSchema[]>()(
+      this,
+      endpoint`${projectId}/job_token_scope/allowlist`,
+      options,
+    );
+  }
 
   addToInboundAllowList<E extends boolean = false>(
     projectId: string | number,
     targetProjectId: string | number,
     options?: Sudo & ShowExpanded<E>,
-  ): Promise<GitlabAPIResponse<AllowListSchema, C, E, void>>;
+  ): Promise<GitlabAPIResponse<ProjectAllowListSchema, C, E, void>> {
+    return RequestHelper.post<ProjectAllowListSchema>()(
+      this,
+      endpoint`${projectId}/job_token_scope/allowlist/${targetProjectId}`,
+      options,
+    );
+  }
 
   removeFromInboundAllowList<E extends boolean = false>(
     projectId: string | number,
     targetProjectId: string | number,
     options?: Sudo & ShowExpanded<E>,
-  ): Promise<GitlabAPIResponse<void, C, E, void>>;
-}
+  ): Promise<GitlabAPIResponse<void, C, E, void>> {
+    return RequestHelper.del()(
+      this,
+      endpoint`${projectId}/job_token_scope/allowlist/${targetProjectId}`,
+      options,
+    );
+  }
+
+  showGroupsAllowList<E extends boolean = false>(
+    projectId: string | number,
+    options?: Sudo & ShowExpanded<E>,
+  ): Promise<GitlabAPIResponse<CondensedGroupSchema[], C, E, void>> {
+    return RequestHelper.get<CondensedGroupSchema[]>()(
+      this,
+      endpoint`${projectId}/job_token_scope/groups_allowlist`,
+      options,
+    );
+  }
 
-export class ProjectJobTokenScopes<C extends boolean = false> extends ResourceJobTokenScopes<C> {
-  constructor(options: BaseResourceOptions<C>) {
-    /* istanbul ignore next */
-    super('projects', options);
+  addToGroupsAllowList<E extends boolean = false>(
+    projectId: string | number,
+    targetGroupId: number,
+    options?: Sudo & ShowExpanded<E>,
+  ): Promise<GitlabAPIResponse<GroupAllowListSchema, C, E, void>> {
+    return RequestHelper.post<GroupAllowListSchema>()(
+      this,
+      endpoint`${projectId}/job_token_scope/groups_allowlist/${targetGroupId}`,
```

Comment:
For add, targetGroupId is a post body parameter, not a url parameter 
https://docs.gitlab.com/ee/api/project_job_token_scopes.html#add-a-group-to-a-cicd-job-token-allowlist

the target is only a url parameter for the remove endpoints